build/room_ensurer/room_ensurer.py

```diff
@@ -95,6 +97,18 @@ def __init__(
         self._is_first_server = own_server_name == self._first_server_name
         self._apis: Dict[str, GMatrixHttpApi] = self._connect_all()
         self._own_api = self._apis[own_server_name]
+
+        self._user_activity_filepath = user_activity_filepath
```

Comment:
Semantically the user activity thing doesn't fit that well here. Maybe we can split it out into a separate tool in the future.